packages/date-picker/src/FilterDateRangePicker/components/DateRangeValidationMessage/DateRangeValidationMessage.spec.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useRef } from "react"
+import { render, screen } from "@testing-library/react"
+import userEvent from "@testing-library/user-event"
+import {
+  DateRangeValidationMessage,
+  DateRangeValidationMessageProps,
+} from "./DateRangeValidationMessage"
+
+const DateRangeInputFieldWrapper = (
+  props: Partial<DateRangeValidationMessageProps>
+): JSX.Element => (
+  <DateRangeValidationMessage
+    status={{
+      dateEnd: "error",
+      dateStart: "caution",
+    }}
+    validationMessage={{
+      dateStart: '"Date from" is close to the submission date.',
+      dateEnd: '"Date to" cannot be earlier than the "Date from" selection.',
+    }}
+  />
+)
+
+describe("<DateRangeValidationMessage />", () => {
+  describe("will render a single validation field message", () => {
+    it("when the consumer has given it a string", () => {
+      render(
+        <DateRangeValidationMessage
+          errorMessageId="error-message-id"
+          status={{
+            dateEnd: "error",
+          }}
+          validationMessage={"Validation message"}
+        />
+      )
+      expect(screen.getByText("Validation message")).toBeVisible()
+    })
+
+    it("when the consumer has given it a node", () => {
+      render(
+        <DateRangeValidationMessage
+          errorMessageId="error-message-id"
+          status={{
+            dateEnd: "error",
+          }}
+          validationMessage={<p>Validation message</p>}
+        />
+      )
+      expect(screen.getByText("Validation message")).toBeVisible()
+    })
+
+    it("when consumer passes validationMessage object but has the same status", () => {
```

Comment:
```suggestion
    it("combines multiple messages with the same status", () => {
```

Something on the lines of that.